core/src/Kokkos_Serial.hpp

```diff
@@ -92,7 +97,12 @@ class Serial {
   //! The size_type typedef best suited for this device.
   typedef HostSpace::size_type  size_type ;
   //! This device's preferred memory space.
+  #ifdef KOKKOS_ENABLE_SICM
+  typedef SICMSpace             memory_space ;
```

Comment:
How does this work if I just do `Kokkos::View<int*,Serial> a("A",N);` is there a default SICMSpace instance?

Reply:
I think there is saw the default constructor - though I am not sure if calling sicm-init repeatedly is good? I.e. in:

```c++
Kokkos::View<int*,Serial> a1("A1",N);
Kokkos::View<int*,Serial> a1("A1",N);
Kokkos::View<int*,Serial> a1("A1",N);
Kokkos::View<int*,Serial> a1("A1",N);
Kokkos::View<int*,Serial> a1("A1",N);

Reply:
`sicm_init` can be called multiple times, but work is only done the first time.